cmd/build/v2/servicecontext.go

```diff
@@ -0,0 +1,38 @@
+package v2
+
+import (
+	"github.com/okteto/okteto/pkg/filesystem"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/repository"
+)
+
+type serviceContextInterface interface {
+	IsCleanBuildContext() bool
+}
+
+type serviceContext struct {
```

Comment:
I'd like to know what this does from the struct. Maybe `SvcContextCleanlinessChecker`

Reply:
eecbe80e10436841929955ebfd994cd41629b530